### PR TITLE
Disconnect network

### PR DESCRIPTION
**Pull request by ChristopherMSpencer**

### CODE REVIEW

File: os_faults/drivers/tcpcloud.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class TCPCloudNodeCollection(node_collection.NodeCollection):
 
     def connect(self, network_name):
         LOG.info("Connect network '%s' on nodes: %s", network_name, self)
-        task = {'osa_network_mgmt': {
+        task = {'fuel_network_mgmt': {
```
Comment (mwturvey) [inline os_faults/drivers/tcpcloud.py:32]: Why the change from osa_netowrk_mgmt to fuel_network_mgmt?  I'm guessing there are some existing functions for fuel that "just work" and aren't really fuel-specific?

**Review by mwturvey — APPROVED** [review]

File: os_faults/drivers/tcpcloud.py (hunk 1 above)
Comment (ChristopherMSpencer) [reply os_faults/drivers/tcpcloud.py:32]: osa_network_mgmt file doesn't exist in os_faults.ansible.modules only fuel_network_mgmt.py does.  so I changed it back to using this and it looks (so far) like it will work with our deployment

**Review by ldcastell — APPROVED** [review]
